src/main/java/rails/game/correct/CorrectionManager.java

```diff
@@ -92,7 +92,8 @@ protected boolean execute(CorrectionAction action) {
     }
 
 
-    public boolean equals(CorrectionManager cm) {
+    public boolean equals(Object object) {
+        CorrectionManager cm = (CorrectionManager) object;
```

Comment:
This is a dangerous assumption. 
You should add a type check here

Reply:
Yes, yes it is. I was working off the assumption that any `equals()` in place were already invalid due to the parameter declaration of `CorrectionManager` but that would have actually caused the default `equals()` to be called vs the override. So yeah, this should be fixed.

Reply:
LOL, this is what is in my current code base:
```
    public boolean equals(Object object) {
        if ( ! (object instanceof CorrectionManager) ) {
            return false;
        }
...
```

Evidently I already fixed it!

Reply:
As Madoar said please fix :)

Reply:
I've already got this in the following commit-- I'll have to rebase and edit that commit to pull this out :(

I looked through the code and don't believe this `equals()` method is actually being executed and looking at code coverage from unit tests it is certainly not. Given that I was just going to sit on it for the next PR given the pain of rebasing.